src/App/Request.php

```diff
@@ -32,8 +32,13 @@
  */
 class Request
 {
-	/** @var string the default possible headers, which could contain the client IP */
-	const ORDERED_FORWARD_FOR_HEADER = 'HTTP_X_FORWARDED_FOR';
+	/**
+	 * A comma separated list of default headers that could contain the client IP in a proxy request
+	 * Beware: This list is ordered
```

Comment:
This comment belongs in the description of the `proxy.forwarded_for_headers` config value. It's unwarranted here since we control the value.